app/views/layouts/application.html.erb

```diff
@@ -29,12 +29,9 @@
         </ul>
       </div>
       </nav>
-      <%# flash.each do |name, message| %>
-      <%#= content_tag :div, message, id: "flash_#{name}" %>
-      <%# end %>
-      <div>
-        <%= flash[:remove_item].html_safe unless flash[:remove_item].blank? %>
```

Comment:
Double check the commit you are looking at 😉 They are still there. I changed it to add the html_safe to all flash messages.